site-src/faq.md

```diff
@@ -66,5 +66,18 @@
   AppProtocol depend on Kubernetes 1.18 (opt-in) or 1.19 (on by default). 
   There are not any other exceptions to the 1.16+ guideline right now.
 
+* **Q: Is SSL Passthrough supported?**
+  A: SSL Passthrough (wherein a Gateway routes traffic with the [Transport
+  Layer Security (TLS)][tls] encryption _intact_ to a backend service instead of
+  terminating it) is supported by the [TLSRoute][tlsroute] API. Note that SSL
+  Passthrough is _not_ supported by the [HTTPRoute][routes] API as it is not
+  possible to route based on HTTP metadata without terminating TLS. See the
+  [TLS Guide][tlsguide] for more details about passthrough and other TLS
+  configurations.
```

Comment:
To go further with @hbagdi's idea, I think it's probably safe to leave HTTPRoute out entirely.
```suggestion
  terminating it) is supported by [TLSRoutes][tlsroute]. See the
  [TLS Guide][tlsguide] for more details about passthrough and other TLS
  configurations.
```